integration_test/busybenchmark.js

```diff
@@ -0,0 +1,18 @@
+let durationSeconds = 600;
```

Comment:
Why isn't it const?

Reply:
Ah, because you modify it. I'd rather pass the duration to the benchmark function as an argument and then `benchmark(process.argv.length > 2 ? process.argv[2] : 600)`.

Reply:
Done.